src/binarylane/console/app.py

```diff
@@ -20,45 +20,41 @@ class App(Runner):
     to run(). The primary function of this class is to invoke the appropriate runner for the supplied arguments.
     """
 
+    HELP_DESCRIPTION: ClassVar[str] = "Display available commands and descriptions"
+
     @dataclass
     class Request:
         command: List[str]
 
-    _parser: Parser
-    _tree: Node
-
     def __init__(self) -> None:
-        context = Context("bl", "bl is a command-line interface for the BinaryLane API")
+        context = Context("", "bl is a command-line interface for the BinaryLane API")
```

Comment:
I couldn't get "bl is a command-line interface for the BinaryLane API" to be displayed, it looks like it is unused, `bl --help`  outputs the value set in `_update_context` :

```python
self._parser.description = f"Access {source.name} commands"
```

```
usage: bl [OPTIONS] COMMAND

Access commands

Options:
```

Reply:
Thanks -  that's a bug,  will fix.